pyopendds/dev/itl2py/PythonOutput.py

```diff
@@ -78,7 +77,12 @@ def get_python_default_value_string(self, field_type):
             elif isinstance(field_type, EnumType):
                 return type_name + '.' + field_type.default_member
             elif isinstance(field_type, SequenceType):
-                return 'field(default_factory=list)'
+                subfield_type = field_type.base_type
+                subtype_name = self.get_python_type_string(subfield_type)
+                if isinstance(subfield_type, PrimitiveType):
+                    return 'field(default_factory=lambda :['+self.primitive_types[subfield_type.kind][1]+'])'
```

Comment:
for more lisibility :
`return 'field(default_factory=lambda :[' + self.primitive_types[subfield_type.kind][1] + '])'`
or
`return "field(default_factory=lambda :[{type}])".format(type=self.primitive_types[subfield_type.kind][1])`

I prefer the 2nd one